modules/reindex/src/test/java/org/elasticsearch/index/reindex/remote/RemoteInfoTests.java

```diff
@@ -19,28 +19,28 @@
 
 package org.elasticsearch.index.reindex.remote;
 
-import org.elasticsearch.index.reindex.RemoteInfo;
 import org.elasticsearch.common.bytes.BytesArray;
+import org.elasticsearch.index.reindex.RemoteInfo;
 import org.elasticsearch.test.ESTestCase;
 
 import static java.util.Collections.emptyMap;
 
 public class RemoteInfoTests extends ESTestCase {
     private RemoteInfo newRemoteInfo(String scheme, String prefixPath, String username, String password) {
-        return new RemoteInfo(scheme, "testhost", 12344, prefixPath, new BytesArray("testquery"), username, password, emptyMap(),
-                RemoteInfo.DEFAULT_SOCKET_TIMEOUT, RemoteInfo.DEFAULT_CONNECT_TIMEOUT);
+        return new RemoteInfo(scheme, "testhost", 12344, prefixPath,new BytesArray("{\"match_all\":{}}"), username, password, emptyMap(),
```

Comment:
I would prefer `new BytesArray("{ \"foo\" : \"bar\" }"`, makes it more explicit what we rely on (JSON and not query-validity).

Reply:
`"{\"match_all\":{}}"` is JSON?

Reply:
Oh, you just just want a nonsense query that is very clearly JSON I think?